pkg/agent/agent.go

```diff
@@ -26,16 +27,20 @@ type Klocksmith struct {
 	nc   v1core.NodeInterface
 	ue   *updateengine.Client
 	lc   *login1.Conn
+	ri   time.Duration
+	rt   time.Duration
```

Comment:
Can you name this reapTimeout or gracePeriod? No need to emulate these (poor imo) existing field names